docs/docusaurus/docs/core/connect_to_data/filesystem_data/_create_a_batch_definition/_tab-file_batch_definition.md

```diff
@@ -42,7 +42,10 @@ Batch Definitions for File Data Assets can be configured to return the content o
 
    <TabItem value="partitioned" label="Partitioned">
    
-   GX Core currently supports partitioning File Data Assets based on dates.  The files can be returned by year, month, or day.
+   GX Core currently supports partitioning File Data Assets based on dates.  The files can be returned by year, month, or day. To define a Batch Definition, you need to provide the following information: 
+
+    - `name`: A name by which you can reference the Batch Definition in the future.  This should be unique within the Data Asset.
+    - `regex`: A regular expression used to match against file names. When using a Filesystem Data Source, `regex` is relative to Data Source param `base_directory`.
```

Comment:
suggestion to use the full word "parameter" for clarity. also adds a missing article

```suggestion
    - `regex`: A regular expression used to match against file names. When using a Filesystem Data Source, `regex` is relative to the Data Source parameter `base_directory`.
```